yarn.lock

```diff
@@ -14077,7 +14077,6 @@ use@^3.1.0:
   version "3.1.1"
   resolved "https://registry.yarnpkg.com/use/-/use-3.1.1.tgz#d50c8cac79a19fbc20f2911f56eb973f4e10070f"
   integrity sha512-cwESVXlO3url9YWlFW/TA9cshCEhtu7IKJ/p5soJ/gGpj7vbvFrAY/eIioQ6Dw23KjZhYgiIo8HOs1nQ2vr/oQ==
-
```

Comment:
Hmm, yea there shouldn't be a change to this file vs main since there's no change to package.json.

When in doubt, you can always run `yarn` to rebuild this file.